tests/wicked/before_test.pm

```diff
@@ -20,7 +20,7 @@ use network_utils qw(iface setup_static_network);
 sub run {
     my ($self, $ctx) = @_;
     $self->select_serial_terminal;
-    if (check_var('WICKED', '2nics')) {
+    if (check_var('WICKED', '2nics') && check_var('IS_WICKED_REF', '0')) {
```

Comment:
What do you think in making this generic. Not in this PR, but in general, that we can access `$ctx->ifaceX()` or `$ctx->iface(X)`

Reply:
only after we will get test case which will use 3 NICs .just for two it would just create additional complexity without paying back much

Reply:
We could avoid the IS_WICKED_REF check.

Reply:
don't see how you would avoid it.  So you can come up with PR and we will discuss :)